modFastVM/test/org/aion/vm/DummyRepository.java

```diff
@@ -121,14 +122,14 @@ public int getStorageSize(Address address) {
     public void addStorageRow(Address addr, ByteArrayWrapper key, ByteArrayWrapper value) {
         Map<String, byte[]> map = storage.computeIfAbsent(addr, k -> new HashMap<>());
 
-        map.put(key.toString(), value.getData());
+        map.put(key.toString(), (value.isZero()) ? value.getData() : value.getNoLeadZeroesData());
     }
 
     @Override
     public ByteArrayWrapper getStorageValue(Address addr, ByteArrayWrapper key) {
         Map<String, byte[]> map = storage.get(addr);
         if (map != null && map.containsKey(key.toString())) {
-            return new ByteArrayWrapper(map.get(key.toString()));
+            return new DataWord(map.get(key.toString())).toWrapper();
```

Comment:
what's the difference of the output between the commit?

Reply:
It's possible that values can be less than 16 bytes because they get truncated. Previously the repo's used to handle this re-sizing but they no longer do, so this just re-aligns to 16 bytes. It is, admittedly, the wrong place to make the change (the caller should be doing this), but it's a very quick change that prevents having to do it in all the calling tests ... and there are plans to eventually phase these dummy repos out

Reply:
I fixed this up and did it properly.